module-utils/unicode/utf8/utf8/UTF8.cpp

```diff
@@ -798,7 +798,7 @@ bool UTF8::isASCIICombination() const noexcept
     constexpr uint8_t secondCharacterFactor = 10;
     for (; i < len; i += 2) {
         int firstCharacter = 0;
-        if (data[i] == '1') {
+        if (len > 3 && data[i] == '1') {
```

Comment:
Considering what I found out from the spec and observation of Quectel eval board, this seems to be just a workaround this particular problem with `100` number. I'm yet to figure out how to methodically distinguish a number consisting from digits from alphanumeric encoded using BSD.

Reply:
Please add comment clearly stating that it is a workaround for this particular case.

Reply:
Done